### ISSUE TITLE
Standard Model - Summary 

### ISSUE THREAD

**Issue by s6nadavi** [JPsiInVR/organisation#71]

**Comment by s6nadavi** [comment]
Here's a link to a nice summary video about our previous SM posts: 

https://www.youtube.com/watch?v=ehHoOYqAT_U&t=103s

**Comment by s6nadavi** [comment]
Well, we've officially made it through all components of the Standard model! 
You have learned about all of the elementary particles that build our universe and we introduced you to the three fundamental interactions, as well as the Higgs mechanism. Since the Standard Model is quite complex and you might want to refresh your knowledge about the previous posts on it, here's a nice summary of all that we showed you so far. It will be quite useful since you will come across the Standard Model and all its constituents every now and then during our Particle Physics adventure.

https://www.youtube.com/watch?v=ehHoOYqAT_U&t=103s
